collections/Lyell/MS_Lyell_50.xml

```diff
@@ -48,25 +42,38 @@
                   <repository>Bodleian Library</repository>
                   <idno type="shelfmark">MS. Lyell 50</idno>
                </msIdentifier>
+               <head>Theological <foreign>quaestiones</foreign>; Gilbert Crispin; Honorius Augustodunensis</head>
                <physDesc>
-                  <p n="composite">Composite: fols. 1-40 || fols. 41-63</p>
+                  <p n="composite">Two MSS. written in Austria or Germany, bound together.</p>
+                  <objectDesc form="codex">
+                     <supportDesc material="perg">
+                        <support>parchment</support>
+                        <extent>63 leaves 
+                              <dimensions unit="mm" type="leaf"><height>220</height><width>150</width></dimensions>
+                           Edges cropped by binder.
+                        </extent>
+                        <foliation>Modern foliation.</foliation>
+                        <collation/>
+                     </supportDesc>
+                  </objectDesc>
+                  <bindingDesc>
+                     <p>Archaizing binding of second half of 15th cent. (not later than 1488), in wooden boards covered with rough white skin, with parchment pastedowns and original place marker of string. Original central leather strap and metal clasp fastening to pin on the upper cover – compare the clasp of MS. Lat. misc. d. 68, also from Admont. On the front cover is a parchment label with the title (15th cent.) <quote>Lucidarius de peccato originali</quote>.</p>
```

Comment:
Include a `<binding>` element as parent of the `<p>`, with dating attributes (e.g. `notAfter="1488"`)